treeherder/extract/__init__.py

```diff
@@ -0,0 +1,8 @@
+import os
+import sys
+
+VENDOR_PATH = __file__.replace("__init__.py", "").rstrip(os.sep)+os.sep+"vendor.zip"
```

Comment:
This is the part that will change and become a PyPi package, correct?  So I'll ignore this.  :)